inception-project/inception
# #1252 Add timeout for external recommender timeout

**What's in the PR**
* The default HTTP timeout is not enough for running the external recommender  for e.g. flair.

**How to test manually**
* Use the external recommender to annotate the text and check whether there are timeout errors.

**Automatic testing**

**Documentation**
